cli/medperf/utils.py

```diff
@@ -508,3 +525,30 @@ def get_cube_image_name(cube_path: str) -> str:
     except KeyError:
         msg = "The provided mlcube doesn't seem to be configured for singularity"
         raise MedperfException(msg)
+
+
+def validate_password(password):
+    # NOTE: if password checks are changed, make sure you modify config.password_policy_msg
```

Comment:
Are these regexs taken from somewhere? if so, we should credit the source.

Reply:
chatgpt